CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ else()
       message("Unknown solver")
 endif()
 
+### Limits designed to make bad updates fail gracefully ###
+set(STRAIN_RATE_LIMIT "1.0e10" CACHE STRING "Limit on strain rate -- NEML may return an error if a strain rate grows beyond this value")
+mark_as_advanced(STRAIN_RATE_LIMIT)
+add_definitions(-DNEML_STRAIN_RATE_LIMIT=${STRAIN_RATE_LIMIT})
```

Comment:
same as above

Reply:
Same thing, I think this is fine as if you're in our CMakeLists.txt you're always configuring NEML options.